addons/mail/static/src/components/activity/tests/activity_tests.js

```diff
@@ -1276,6 +1276,52 @@ QUnit.test('data-oe-id & data-oe-model link redirection on click', async functio
     );
 });
 
+QUnit.test('chatter is updated when activity type is changed', async function (assert) {
+    assert.expect(2);
+
+    const activityId = 513;
+    this.data['res.partner'].records.push({
+        activity_ids: [activityId],
+        id: 100,
+    });
+    this.data['mail.activity'].records.push({
+        activity_category: 'upload_file',
```

Comment:
Ideally the mock server should take care of returning this related field properly from the activity type itself, is it actually needed to define it again?